test/integration/joint_position_controller_system.cc

```diff
@@ -207,3 +207,80 @@ TEST_F(JointPositionControllerTestFixture,
 
   EXPECT_NEAR(targetPosition, currentPosition.at(0), TOL);
 }
+// Tests that the JointPositionController accepts joint position
+// sub_topic commands
+TEST_F(JointPositionControllerTestFixture,
+       GZ_UTILS_TEST_DISABLED_ON_WIN32(
+        JointPositonMultipleJointsSubTopicCommand))
+{
+  using namespace std::chrono_literals;
+
+  // Start server
+  ServerConfig serverConfig;
+  const auto sdfFile = std::string(PROJECT_SOURCE_PATH) +
+    "/test/worlds/joint_position_controller_multiple_joints_subtopic.sdf";
```

Comment:
`gz::common::joinPaths`

Reply:
I edited it also in all the rest of the file from the previous method to the now preferred joinPaths method.